### ISSUE TITLE
Can't input numbers on mobile Safari

### ISSUE THREAD

**Issue by stx** [Tkko/Flutter_Pinput#128]
v2.2.31

Repro code: `Pinput()`

Visit webpage using Safari on iPhone, try to tap numbers, no input.

**Comment by stx** [comment]
Yep, it works there. Our app is using Flutter stable 3.7.5. What was the demo page built with?

**Comment by Tkko** [comment]
I've just pushed the update to the demo page and it's built with flutter 3.7.6

**Comment by stx** [comment]
@Tkko Found the problem - the SelectionArea widget. Full repro:

```
void main() async {
  runApp(
    MaterialApp(
      home: SelectionArea(
        child: Scaffold(
          body: Pinput(),
        ),
      ),
    ),
  );
}
```

**Comment by stx** [comment]
Wrapping `Pinput()` with `SelectionContainer.disabled()` to disable selection on it doesn't work either.
